ArduPlane/quadplane.cpp

```diff
@@ -2131,9 +2131,15 @@ void QuadPlane::vtol_position_controller(void)
           than the actual velocity curve (for a high drag
           aircraft). Nose down will cause a lot of downforce on the
           wings which will draw a lot of current and also cause the
-          aircraft to lose altitude rapidly.
+          aircraft to lose altitude rapidly.pitch limit varies also with speed
+          to prevent inability to progress to position if moving from a loiter
+          to landing
          */
-        float pitch_limit_cd = linear_interpolate(-300, plane.aparm.pitch_limit_min_cd,
+        float minlimit = linear_interpolate(-300, aparm.angle_max,
+                                            (double)ahrs.groundspeed(), 
+                                            wp_nav->get_default_speed_xy() * 0.03, 
+                                            wp_nav->get_default_speed_xy() * 0.01);
```

Comment:
I'm not sure you've actually tested this, since the var low/var high parameters here are out of order, the last 2 arguments are supposed to be var_low, var_high, while you are providing them in the opposite order, which means the interpolate doesn't run correctly.

Reply:
as I said...I dont know how to test in MP SITL....
thanks, will change order......anything else that's obvious?